### PR TITLE
don't use self-built yard

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: dependencies/buster/puppet-strings/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-puppet-strings (1.1.0-2) stable; urgency=low
```
Comment (ekohl) [inline dependencies/buster/puppet-strings/changelog:1]: I don't see any change here (same for focal)

Comment (evgeni) [reply dependencies/buster/puppet-strings/changelog:1]: that's correct, it was already using the "real" package there, but I wanted the versions to align.

Comment (evgeni) [reply dependencies/buster/puppet-strings/changelog:1]: my plan is to get rid of most contents in the focal and bionic folders, unless the builds *really* differ from buster, but for that they need to match first ;)

Comment (ekohl) [reply dependencies/buster/puppet-strings/changelog:1]: Fair enough.

**Review by ekohl — APPROVED** [review]

**Pull request merged**